packages/beacon-node/src/network/peers/peerManager.ts

```diff
@@ -657,16 +657,26 @@ export class PeerManager {
    */
   private onLibp2pPeerDisconnect = (evt: CustomEvent<Connection>): void => {
     const {direction, status, remotePeer} = evt.detail;
+    const peerIdStr = remotePeer.toString();
+
+    // prevent automatic/immediate reconnects
+    const coolDownMin = 60;
+    this.peerRpcScores.applyReconnectionCoolDown(remotePeer, coolDownMin);
```

Comment:
I think libp2p will call this method for both inbound and outbound. For outbound, we already treated it in `goodbyeAndDisconnect()`

this `applyReconnectionCoolDown` should only be applied for `inbound direction` only. Ideally we need to analyze good bye code like in `onGoodbye` but not all clients implement that (for example Nimbus) so we have to a assume a good `*BEFORE_DECAY_MS` value for it